.github/workflows/docker.yml

```diff
@@ -0,0 +1,31 @@
+name: Build and Push Docker Image
+
+on:
+  push:
+    tags:
+      - 'v*'
```

Comment:
should it be triggered only when new tag is pushed?

Reply:
This is what we decided to do for now. So with this setup there won't be docker images between releases.